org-gtd-areas-of-focus.el

```diff
@@ -38,9 +38,9 @@
 
 (defcustom org-gtd-areas-of-focus '("Home" "Health" "Family" "Career")
   "The current major areas in your life where you don't want to drop balls."
-  :type 'list
   :group 'org-gtd-horizons
```

Comment:
I removed the `defgroup` I added here as well and changed the `:group` argument value back to what it was as well as I assume the same argument you made for similar changes applies here too.

Reply:
That's correct, areas of focus are technically one of the horizons.

Reply:
Raised as issue #159.